superset/views/core.py

```diff
@@ -652,13 +636,6 @@ def save_or_overwrite_slice(  # noqa: C901
                     _("You don't have the rights to alter this dashboard"),
                     status=403,
                 )
-
-            flash(
-                _("Chart [{}] was added to dashboard [{}]").format(
```

Comment:
tested that chart adding to dashboard is handled frontend now and flashed properly without these obsolete flash messages